infrastructure/src/main/java/org/corfudb/infrastructure/AutoCommitService.java

```diff
@@ -155,16 +160,9 @@ public synchronized void runAutoCommit() {
                     break;
                 }
 
-                // When inspecting address, log unit server could throw a TrimmedException if
-                // the address to inspect is trimmed, then we need to adjust committed tail.
-                if (re instanceof TrimmedException) {
-                    trimmed = true;
-                }
-
-                if (re instanceof NetworkException || re.getCause() instanceof TimeoutException) {
+                if (re.getCause() instanceof TimeoutException) {
```

Comment:
This change will break the retry logic.
Let's say retryNum is 3 and current iteration is 3, then it hit a TrimmedException. 
It will simply mark trimmed = true, then finish the while loop. It won't print logs `runAutoCommit: retry exhausted`

Reply:
Thanks! I have revert the changes!